dojo/notifications/helper.py

```diff
@@ -61,7 +61,8 @@ def create_notification(event=None, *args, **kwargs):
 
         # only send to authorized users or admin/superusers
         if product:
-            users = users.filter(Q(id__in=product.authorized_users.all()) | Q(is_superuser=True) | Q(is_staff=True))
+            qs = (product.authorized_users.all() | product.prod_type.authorized_users.all())
+            users = users.filter(Q(id__in=qs) | Q(is_superuser=True) | Q(is_staff=True))
```

Comment:
why not keep the same construct as on the left?
```
users = users.filter(Q(id__in=product.authorized_users.all()) | Q(id__in=productprod_type.authorized_users.all()) | Q(is_superuser=True) | Q(is_staff=True))
```

Reply:
A grab at brevity?

Reply:
hmm the reason I ask is because when reading the code it makes me think that there is something special going on, some kind of construct to make it work. but after reading it a couple of times I realize it's just filtering the same as the rest.
so from that perspective I'd think it's better to have it the same so readers don't have to think extra.